src/main/java/uk/nhs/digital/nhsconnect/nhais/model/edifact/PersonName.java

```diff
@@ -126,4 +125,15 @@ public void preValidate() throws EdifactValidationException {
     @Override
     protected void validateStateful() throws EdifactValidationException {
     }
+
+    public enum PatientIdentificationType {
+        OPI("Official Patient Identifier"),
```

Comment:
Should be the other way around OFFICIAL_PATIENT_IDENTIFIER("OPI")

1) "description" string services no purpose here
2) if someone were to refactor the enum names it would break messaging